File: Atividade_Extra-2/Questao5/ControleDeAudio.js
var ControleDeAudio = /** @class */ (function () {
    function ControleDeAudio() {
        this.volume = 2;
    }
    ControleDeAudio.prototype.aumentarVolume = function () {
        if (this.volume === 10) {
            console.log("O volume já está no maxímo!");
            return;
        }
        this.volume++;
        console.log("\nVolume Aumentado!");
        console.log("Volume Atual: ".concat(this.lerVolume()));
    };
    ControleDeAudio.prototype.diminuirVolume = function () {
        if (this.volume === 0) {
            console.log("O volume já está no minímo!");
            return;
        }
        this.volume--;
        console.log("\nVolume Diminuido!");
        console.log("Volume Atual: ".concat(this.lerVolume()));
    };
    ControleDeAudio.prototype.lerVolume = function () {
        return this.volume;
    };
    return ControleDeAudio;
}());
function main() {
    console.clear();
    var myAudio = new ControleDeAudio();
    console.log("Volume Inicial: ".concat(myAudio.lerVolume()));
    myAudio.aumentarVolume();
    myAudio.diminuirVolume();
}
main();

File: Atividade_Extra-2/Questao7/ControleDeAudio.java
public class ControleDeAudio {
    private int volume = 2;

    public void aumentarVolume() {
        if (volume == 10) {
            System.out.println("O volume já está no máximo!");
            return;
        }
        volume++;
        System.out.println("\nVolume Aumentado!");
        System.out.println("Volume Atual: " + lerVolume());
    }

    public void diminuirVolume() {
        if (volume == 0) {
            System.out.println("O volume já está no mínimo!");
            return;
        }
        volume--;
        System.out.println("\nVolume Diminuído!");
        System.out.println("Volume Atual: " + lerVolume());
    }

    public int lerVolume() {
        return volume;
    }

    public static void main(String[] args) {
        ControleDeAudio myAudio = new ControleDeAudio();
        System.out.println("Volume Inicial: " + myAudio.lerVolume());

        myAudio.aumentarVolume();
        myAudio.diminuirVolume();
    }
}
